auth-microservice/Pipfile

```
@@ -19,6 +19,7 @@ uvicorn = "*"
 python-dotenv = "*"
 bcrypt = "*"
 factory-boy = "*"
+auth-validator = {editable = true, path = "/Users/charlyjazz/Charlyjazz/Chat-Firebase-Apache-Camel/auth-validator"}
```

This won't work outside of that specific install. To make it portable you should consider using the repo as the source instead. According to [this SO answer](https://stackoverflow.com/a/19516714) it should be something like this:
```suggestion
auth-validator = {editable = true, path = "git+https://github.com/CharlyJazz/Chat-Firebase-Apache-Camel.git/#egg=pkg&subdirectory=auth-validator"}
```